api/src/main/java/io/minio/MinioClient.java

```diff
@@ -2102,12 +2102,15 @@ public void copyObject(String bucketName, String objectName, String destBucketNa
   }
 
   /**
-   * Returns an presigned URL to download the object in the bucket with given expiry time with custom request params.
+   * Returns an presigned URL string for given method of given object in the bucket with given expiry time with custom
+   * request params.
```

Comment:
suggestion for readability:
Generates a presigned URL string with given HTTP method, expiry time and custom request params for a specific object in the bucket

Reply:
slightly modified